Lib/gftools/scripts/gen_debg_deps.py

```diff
@@ -0,0 +1,34 @@
+"""
+gftools gen-debg-deps
+Add the dependencies used to generate the font to the font's Debg table
+
+Usage:
+
+gftools gen-debg-deps font.ttf -o font-deps.ttf
+gftools gen-debg-deps font.ttf --inplace
+"""
+from argparse import ArgumentParser
+from gftools.builder.dependencies import requirements_to_debg_table
+from fontTools.ttLib import TTFont
+
+
+def main(args=None):
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument("font", help="Path to font")
+    parser.add_argument("--inplace", help="Overwrite font", action="store_true")
+    parser.add_argument("-o", "--out", help="Output path for new font")
```

Comment:
this should be a "mutually exclusive group" of arguments:
https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group